bin/i18n/sync-codeorg-all

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env ruby
+require_relative '../../lib/cdo/github'
+require_relative 'i18n_script_utils'
+
+IN_UP_BRANCH = "i18n-sync-in-up-#{Date.today.strftime("%m-%d")}"
+DOWN_OUT_BRANCH = "i18n-sync-down-out-#{Date.today.strftime("%m-%d")}"
+
+def should_i(question)
+  loop do
+    print "Should I #{question}? [Yes]/Skip/Quit: "
+    response = gets.strip.downcase
+    puts ''
+    if 'yes'.start_with?(response) # also catches blank/return ;)
+      return true
+    elsif 'skip'.start_with?(response) || 'no'.start_with?(response)
+      return false
+    elsif 'quit'.start_with?(response)
+      puts "quitting"
+      exit(-1)
+    else
+      puts "Sorry, I didn't understand that.\n\n"
+    end
+  end
+end
+
+def update_repo
+  return unless should_i "update the repo"
+  `git checkout staging`
+  `git pull origin staging`
+end
+
+def sync_in
+  return unless should_i "sync in"
+  run_standalone_script(File.join(__dir__, "sync-codeorg-in"))
+end
+
+def sync_up
+  return unless should_i "sync up"
+  run_standalone_script(File.join(__dir__, "sync-codeorg-up"))
+end
+
+def create_in_up_pr
+  return unless should_i "create the in & up PR"
+  `git checkout -B #{IN_UP_BRANCH}`
+
+  `git add -u dashboard/config/locales/*.en.yml`
+  `git add -u i18n/locales/source/dashboard`
+  `git commit -m "dashboard i18n sync"`
```

Comment:
This (some number of `git add`, then `git commit`) seems like a common pattern in this file. Perhaps worth extracting to a method `git_add_and_commit(array_of_paths, commit_message)`.